data/profiles/juice.profile

```diff
@@ -0,0 +1,123 @@
+{
+  "additional_scripts": [
+    ""
+  ],
+  "assets": [
+    "base",
+    "scene/solarsystem/missions/juice/fieldlines",
+    "scene/solarsystem/missions/juice/fov/janus",
```

Comment:
A lot of them are not looking correct since the mission is still early and there is some more work to be done later.  In particular, we don't currently support circular FOVs in the RenderableFOV class and they would throw errors when loading the profile, so I decided to only include the imaging instruments Janus and NavCam, just as we did with New Horizons and Rosetta

Reply:
Added a comment in the description of the profile for that